CFC_WebApp/clients/leaderboard/SQL.py

```diff
@@ -0,0 +1,23 @@
+import pyodbc
+
+CONNECTION = pyodbc.connect("DRIVER={SQL Server};SERVER=tcp:xalxu8zk56.database.windows.net,1433;DATABASE=e-mission_db;Uid=joshz@xalxu8zk56;PWD=Nathan123")
```

Comment:
Are we retaining this file?
If so, shouldn't it support both Azure and Couchbase with an abstraction layer on top?